### ISSUE TITLE
Toonin on quasar's mobile SPA lacks aesthetic, usability, and accessibility 

### ISSUE THREAD

**Issue by ArsalaBangash** [grey-software/toonin#190]
![image](https://user-images.githubusercontent.com/19757203/80683911-e92b2a80-8a92-11ea-8ef8-a967d0852e63.png)

- [x] Pull tabs out to the nav bar or bottom bar
- [x] Replace q-card as the parent container for Toonin tabs 
- [x] Disable the Share tab on mobile. Query for a mobile device using 
``` 
<div v-if="$q.platform.is.mobile">
I'm only rendered on mobile!
</div>
```
